ipfsspec/core.py

```diff
@@ -166,7 +174,23 @@ def ls(self, path, detail=True, **kwargs):
         logger.debug("ls on %s", path)
         res = self._gw_apipost("ls", arg=path)
         links = res["Objects"][0]["Links"]
-        types = {1: "directory", 2: "file"}
+
+        # Documentation for the type codes doesn't seem to have a great location.
+        # Currently 2023-10-17, the ls API docs specify the ls return
+        # structure, but not what the type integers mean:
+        # http://docs.ipfs.tech.ipns.localhost:8080/reference/kubo/rpc/#api-v0-ls
```

Comment:
For documentation, I'd generally prefer links which are publicly reachable, also without a running local gateway.
```suggestion
        # http://docs.ipfs.tech/reference/kubo/rpc/#api-v0-ls
```

Reply:
woops didn't notice that.